foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -66,11 +66,20 @@
             ]
           }
         ]
+      },
+      {
+        "name": "data-lineage-package-import-migration",
```

Comment:
S: Add to the automatic migrations table in `DRAFT_RELEASE_NOTES` so users know how to shut this off if they don't want it.